array_binary_search/array_binary_search.js

```diff
@@ -0,0 +1,33 @@
+'use strict';
+
+function findMidpoint(start, end) {
+  return Math.ceil(start + ((end - start) / 2));
```

Comment:
"Without utilizing any of the built-in methods available to your language"  <- Be careful about this